website/docs/API.md

```diff
@@ -803,6 +805,10 @@ This option is also available as `defaultHidden` alias for compatibility with [R
 
 Default [debug options](#debug) to be used when calling `debug()`. These default options will be overridden by the ones you specify directly when calling `debug()`.
 
+#### `hostComponentNames` option
+
+The internal names of the components `Text` and `TextInput` used by `byText`, `byDisplayValue` and `byPlaceholderText` queries. This should only be changed if you're using a version of React Native that is not compatible with the version of @testing-library/react-native. If that's the case, then failed getBy and findBy queries should suggest the appropriate config to use.
```

Comment:
How about tweaking it a bit to be less about implemeantion details but more from user's POV

```suggestion
Specifies the host component names RNTL expects React Native to use when rendering some built-in component types (e.g. `Text` and `TextInput`).

You normally won't need to change this option. In the rare occasion when React Native would change the relevant host components names, RNTL will show you a warning suggesting proper values to use.
```

Reply:
Agreed, this is way clearer !